.github/workflows/testBuildHybrid.yml

```diff
@@ -63,6 +59,48 @@ jobs:
           echo "REF=$(gh pr view ${{ github.event.inputs.PULL_REQUEST_NUMBER }} --json headRefOid --jq '.headRefOid')" >> "$GITHUB_OUTPUT"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+  getOldDotPR:
+    runs-on: ubuntu-latest
+    needs: validateActor
+    if: ${{ fromJSON(needs.validateActor.outputs.READY_TO_BUILD) }}
+    outputs:
+      OLD_DOT_PR: ${{ steps.old-dot-pr.outputs.result }}
+    steps:
+      - name: Check if author specifed Old Dot PR
+        id: old-dot-pr
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ github.token }}
+          result-encoding: string
+          script: |
+            const pullRequest = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: env.PULL_REQUEST_NUMBER,
```

Comment:
We need to use `process.env` here otherwise the env is not defined. Fixing it here https://github.com/Expensify/App/pull/55577